frontend/src/domains/admin/CreateRoomModal/CreateRoomModal.tsx

```diff
@@ -22,20 +27,29 @@ export default function CreateRoomModal({
   onClose,
 }: CreateRoomModalProps) {
   const theme = useAppTheme();
-  const [roomName, setRoomName] = useState('');
+  const [organizationName, setOrganizationName] = useState('');
+  const [disabled, setDisabled] = useState(true);
 
   const { selectedCategories, handleCategoryClick, handleCategoryTagClick } =
     useCategorySelection();
 
+  const { createRoom, isPending } = useCreateOrganization({
+    onClose,
+  });
+
+  useEffect(() => {
+    setDisabled(!organizationName || selectedCategories.length === 0);
+  }, [organizationName, selectedCategories]);
```

Comment:
```suggestion
  const {disabled} = useSubmitDisabled({organizationName, selectedCategories})
```
위와 같이 useEffect와 useState를 `useSubmitDisabled`훅 안에 숨기는 거 어떤가요??

Reply:
ㅎㅎ 흐린눈하고 지나갔는데 딱들켰네요 :)
수정했습니다~

[feat: useSubmitDisabled 훅 추가](https://github.com/woowacourse-teams/2025-feed-zupzup/pull/560/commits/e1874b70ce511024c3271dfa0575dbc759ae64cd)